tock/employees/admin.py

```diff
@@ -1,11 +1,16 @@
 from django.contrib import admin
 from django.forms.models import BaseInlineFormSet
 
-from .models import UserData
+from .models import UserData, EmployeeGrade
 
-class EmployeeDataAdmin(admin.ModelAdmin):
+class UserDataAdmin(admin.ModelAdmin):
     list_display = ('user',)
     search_fields = ('user__username',)
 
 
-admin.site.register(UserData)
+class EmployeeGradeAdmin(admin.ModelAdmin):
+    pass
+
```

Comment:
No real need, mostly as a placeholder in case we wanted to customize the admin interface for EmployeeGrade. Happy to excise it as dead code.